components/automate-ui/src/app/components/settings-sidebar/settings-sidebar.component.html

```diff
@@ -1,4 +1,4 @@
-<chef-sidebar [attr.major-version]="iamMajorVersion$ | async">
+<chef-sidebar>
```

Comment:
FWIW: That mechanism was for use by Cypress tests, which I had forgotten about. Might be less work and less error-prone to use that (though moved to the top level app file) instead of having to set the cypress version in many places. Just a thought.